src/libraries/System.Net.Mail/Directory.Build.props

```diff
@@ -3,6 +3,5 @@
   <PropertyGroup>
     <StrongNameKeyId>Open</StrongNameKeyId>
     <IncludePlatformAttributes>true</IncludePlatformAttributes>
-    <UnsupportedOSPlatforms>browser</UnsupportedOSPlatforms>
```

Comment:
Perhaps, it will require annotating the respective classes/methods that haven't enabled on browser.

Reply:
How is that done?  And maybe an example of what you are referring to please.  Thanks Maxim

Reply:
For example, please see https://github.com/dotnet/runtime/pull/41963. There might a lot of work. 
@mdh1418 Do you happen to know if we still have to annotate all the classes/method that haven't been enabled on browser?

Reply:
AFAIK we should still annotate classes/methods that are not supported on browser for the Platform Compatibility Analyzer to know which APIs to throw warnings. Another example of annotating the unsupported attribute at the class/method level: https://github.com/dotnet/runtime/pull/41891

Currently there is an effort to get the analyzer to throw warnings for Browser https://github.com/dotnet/runtime/issues/42335

Reply:
So if I decorate the SmtpClient class as follows:

```
    [UnsupportedOSPlatform("browser")]
    public class SmtpClient : IDisposable
    {
```

for the class that will be for the whole class and will not need to to decorate each method correct?

Reply:
yes, it should be so